Source/MDFastBinding/Private/MDFastBindingFunctionWrapper.cpp

```diff
@@ -35,18 +37,18 @@ UClass* FMDFastBindingFunctionWrapper::GetFunctionOwnerClass() const
 	return OwnerClassGetter.IsBound() ? OwnerClassGetter.Execute() : nullptr;
 }
 
-const TArray<const FProperty*>& FMDFastBindingFunctionWrapper::GetParams()
+TArray<const FProperty*> FMDFastBindingFunctionWrapper::GetParams()
 {
 #if WITH_EDITORONLY_DATA
-	if (!LastFrameFunctionUpdated.IsSet() || LastFrameFunctionUpdated.GetValue() != GFrameCounter)
+	if (!LastFrameFunctionUpdated.IsSet() || LastFrameFunctionUpdated.GetValue() != GFrameCounter || FunctionMember.ResolveMember<UFunction>() != FunctionPtr)
```

Comment:
Is this extra condition not needed in the other functions? (GetFunctionPtr, CallFunction)

Reply:
The main offender was `GetParams` as it was the only one being called during `SetupBindingItems`, but I see no harm in adding this check for the other functions if you feel it would make things more robust.

Reply:
Feel like we might as well have it consistent throughout

Reply:
Resolved in commit [927b7bc](https://github.com/DoubleDeez/MDFastBinding/pull/40/commits/927b7bc99fbcb2df4c1c3398a0b947737532edc0)